src/pages/create.tsx

```diff
@@ -211,54 +220,58 @@ const CreateProgramPage: Page = () => {
             onSrcChange={setProgramLogoURL}
           />
         </div>
+        <form className="space-y-10">
```

Comment:
nit: Personally I'd favor using `div` spacers over `space-y-10` but up to u

Reply:
I personally like using space when there are more than like 2 or 3 HTML elements because it reduces the amount of code that is written :0